packages/server/tests/utils/TestUtils.ts

```diff
@@ -385,18 +417,23 @@ export function initTestUtils(): TestUtilsReturnedValue {
         return socket;
     }
 
-    async function createUserForSocketConnection({
+    /**
+     * This function performs an api token authentication
+     */
+    async function createAuthenticatedUserForSocketConnection({
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** 👍